osgp/shared/shared/src/main/java/org/opensmartgridplatform/shared/security/RsaEncryptionService.java

```diff
@@ -234,7 +235,7 @@ private KeyPair createKeyPair(final PublicKey publicKey, final PrivateKey privat
         throw new EncrypterException(PUBLIC_KEY_DOES_NOT_BELONG_WITH_PRIVATE_KEY);
     }
 
-    private RSAPublicKey getRsaPublicKey(final PublicKey publicKey) {
+    private RSAPublicKey getRsaPublicKey(final PublicKey publicKey) throws EncrypterException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'org.opensmartgridplatform.shared.exceptionhandling.EncrypterException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)